docker/docker_image_dest.go

```diff
@@ -61,41 +60,36 @@ func (d *dockerImageDestination) PutManifest(manifest []byte) error {
 	defer res.Body.Close()
 	if res.StatusCode != http.StatusCreated {
 		body, err := ioutil.ReadAll(res.Body)
-		if err == nil {
-			logrus.Debugf("Error body %s", string(body))
+		if err != nil {
+			return err
```

Comment:
I think we should be reporting the HTTP status code, instead of the error reading the rest of the body, here.